### DISCUSSION TITLE
Custom keymap for specific command

### DISCUSSION THREAD

**Discussion by jesus-333** [kylechui/nvim-surround#206] [Q&A] [answered]
Premising that I am not a lua expert or a Vim expert... I start some months ago to use neovim and I loved it. Now I'm trying to create my costum keymaps for all the plugin that I have installed.

I tried to remap some of the commands that I use more often to simple combination of key. 
For example I tried:
```lua
keymap("n", "((", "ysiwb", opts)
```
to add the round brackets around a word but it doesn't work. I'd like to add similar commands also for other symbols (e.g. `""`  to surround something with " or `))` to remove the round brackets

**Comment by kylechui** [comment, marked as answer]
Assuming you are using `vim.keymap.set`, the `remap` flag is set to `false` by default, so Neovim is interpreting `ysiwb` in terms of built-in commands, having no knowledge of `nvim-surround` bindings. You want to set `remap` to true, i.e.
```lua
vim.keymap.set("n", "((", "ysiwb", { remap = true })
```

See `:h vim.keymap.set` and `:h remap` for more details.

**Reply by jesus-333** [reply]
It works perfectly!

**Comment by smitropoulos-arista** [comment]
Very nice! I think this deserves to go into the readme?
